src/renderer/Components/ReceivingPackage/index.jsx

```diff
@@ -13,8 +14,12 @@ import { RECEIVING_ALERT, COMMON_ALERT } from "@renderer/constants/messages";
 function ReceivingPackage() {
   const [currentPackage, setCurrentPackage] = useState([]);
   const { setInfoMessage, openInfoModal } = usePackageStore();
+  const [searchParams, setSearchParams] = useSearchParams();
+  const target = useRef();
```

Comment:
해당 target 이라는 변수명 보다 좀 더 직관적으로 알아볼 수 있는 다른 변수명 targetButton ? 같은게 좋을 것 같습니다.😁
```suggestion
  const targetButton = useRef();
```

Reply:
저도 조금 고민이 있던 부분이었는데 명확성을 위해서 targetButton으로 변경해보겠습니다!👍